terminate called after throwing an instance of 'std::runtime_error'
  what():  Variable non déclarée
Aborted (core dumped)

exit status: 134
